scripts/assets/commands/balena-idle

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -e
+
+usage() {
+	cat <<EOUSAGE
+This script basically idles to keep the container running, while periodically printing to the logs that "Idling...".
+
+Options:
+  -m	Message to print when ilding (Default to "Idling...").
```

Comment:
perhaps we should add an option like `--silent` which doesn't log.

Reply:
updated.